common-npm-packages/webdeployment-common-v2/xdttransformationutility.ts

```diff
@@ -0,0 +1,136 @@
+import tl = require('azure-pipelines-task-lib/task');
+import trm = require('azure-pipelines-task-lib/toolrunner');
+import path = require('path');
+
+export function expandWildcardPattern(folderPath: string, wildcardPattern : string) {
+    var matchingFiles = tl.findMatch(folderPath, wildcardPattern,   { followSymbolicLinks: false, allowBrokenSymbolicLinks: false, followSpecifiedSymbolicLink: false });
```

Comment:
It seems that this line has been changed. Could you please describe why should it be changed?

Reply:
@egor-bryzgalov I have copied this file from master branch and so should be consistent with master branch. My assumption was that master branch will have latest changes.

Reply:
Yeah, it looks like this file has been changed on `master` but not on `common-packages-preview`. I guess it's fine to use the version from `master` since we're creating a new package anyway. What do you think, @egor-bryzgalov?